http/src/test/scala/org/http4s/blaze/http/http2/WriteControllerImplSpec.scala

```diff
@@ -0,0 +1,149 @@
+package org.http4s.blaze.http.http2
+
+import java.nio.ByteBuffer
+
+import org.http4s.blaze.http.http2.mocks.MockTools
+import org.http4s.blaze.pipeline.TailStage
+import org.specs2.mutable.Specification
+
+import scala.collection.mutable
+import scala.concurrent.{Future, Promise}
+
+class WriteControllerImplSpec extends Specification {
+
+  private def mockData(start: Int): ByteBuffer =
+    ByteBuffer.wrap {
+    (0 until (start + 1)).map(_.toByte).toArray
```

Comment:
`(0 to start)`?

Reply:
`start` is an awful name. It needs to be at least size 1 or else it gets filtered. I've just changed it to `i`.